src/plugins/appeals/unbanThread.ts

```diff
@@ -0,0 +1,35 @@
+import {useEvent} from "../../hooks";
+import {Interaction, Message} from "discord.js";
+
+useEvent("interactionCreate", async (interaction: Interaction) => {
+    if (!interaction.isButton()) {
+        return;
+    }
+    const {customId} = interaction;
```

Comment:
Unnecessary destructuring assignment; the custom ID is only accessed once so there is little benefit in defining a variable for it

Reply:
Understandable. Removed the {customId} const and fixed where the const would be necessary.